sql/topology/nodeNetwork.sql

```diff
@@ -201,8 +201,12 @@ BEGIN
     vst_line_locate_point := 'st_linelocatepoint';
   END IF;
 
+  create schema if not exists temp_nodenetwork;
```

Comment:
An extension can not be creating schemas.
Also an extension can not be creating permanent tables